### PR TITLE
Professional equality

### PR DESCRIPTION
**Pull request by nitech-code-of-conduct-suggestions-2**
This isn't the space for these discussions, if there is one, and is destructive to those who have had to defend themselves professionally against this type of discrimination

### CODE REVIEW

**Comment by andrewbolster** [comment]
Potential structural duplicate of #9 ? I'd recognise statements of professional inequality to be "exclusionary"

**Comment by mauricerkelly** [comment]
My feeling on this PR is that it's just too wordy. This is a code of conduct, and not a legal document, and the language used needs to be accessible.

**Comment by andrewbolster** [comment]
Ok, @mauricerkelly do we have a CoC for managing "Close"s?

**Comment by mauricerkelly** [comment]
Maybe I should create a new repo for that CoC 😉

We probably need to get some more admin opinions on this.

**Review by mauricerkelly — CHANGES_REQUESTED** [review]
I'm asking for changes to this because I feel like it's too much like text from a legal document. The preceding examples of harassment are much more natural and accessible to readers. We want people to understand the CoC and not gloss over sections because the language is opaque.

**Review by MarkXA — COMMENTED** [review]
I feel this probably needs to be moved out of "Harassment" (which is generally descriptive of behaviour targeted at individuals) and into a separate section describing general unacceptable behaviour (such as spamming).

**Comment by nitech-code-of-conduct-suggestions-2** [comment]
mo, did you read this after the changes, or is it still too legal?

**Review by mauricerkelly — APPROVED** [review]

**Comment by mauricerkelly** [comment]
@nitech-code-of-conduct-suggestions-2 I'm good with the changes

@MarkXA There are a few things that you've suggested be moved out of "Harassment" - how about we get these things merged in as they stand, and then move them en-masse to a new section?

**Comment by MarkXA** [comment]
Makes sense. I'll raise a fresh PR once they're in.

**Comment by mauricerkelly** [comment]
Can we get some (dis/)approvals on this PR to keep it moving?

**Review by clairebones — APPROVED** [review]
